Xamarin.Forms.Platform.Android/VisualElementTracker.cs

```diff
@@ -60,8 +78,14 @@ protected virtual void Dispose(bool disposing)
 
 			if (disposing)
 			{
-				SetElement(_element, null);
+				if (_loaded)
+				{
+					_element?.SendUnloaded();
+					if (_element is IPageController page && !Application.Current.UseLegacyPageEvents)
```

Comment:
this could just call *HandleViewDetachedFromWindow*

it's the same code plus it sets loaded to false whereas the code here doesn't set loaded to false which might lead to double unloaded events?